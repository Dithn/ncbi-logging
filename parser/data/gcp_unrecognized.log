"1591062496651000","","","","PUT","","200","","","","","","GCS Lifecycle Management","02478f5fe3deaa079d1a4f6033043a80","PUT_Object","sra-pub-src-3","SRR1269314/s_3_11000.p1_export.txt.gzb_fixed.bam"
"1591118933830501","35.202.252.53","1","","GET","/sra-pub-run-8/SRR10303547/SRR10303547.1?GoogleAccessId=data-access-service%40nih-sra-datastore.iam.gserviceaccount.com&Expires=1591478828&userProject=nih-sra-datastore&Signature=ZNzj62MP4PWwSVvtkmsB97Lu33wQq4cFGLyWRJTcb%2F8h1BvVXi3lokOoT16ihScR%0At2EHti%2FgQ80VVMv9BGpAY%2FQ9HTqXeq57N53tMcjXQQMKFVttyXgIW89OLWO0UC0h%0AZdFq5AcKZywgnZql8z3RoaQi%2FPKdrdMO803tW%2Bxe%2Boy8sCd%2FyCXcG9jBrkGbdqPc%0A3xnyuycW1Va4LHIh4muGGdFSIqBk7oaLjkjLV54L8e4InzFMD3Kx0Q5raIlNadxx%0AIX%2B2hoPJuCdSh6IxEikrvUri%2Fd6i9Nqo%2BkZ%2BPSGtvlah9I9AafXrs3EAlwZkvc%2Bp%0AnjssKH8zalZQ5SmPpfHImQ%3D%3D%0A","206","0","262144","31000","storage.googleapis.com","","","AAANsUnhN-04LMubyLe-H4MQzIYbbqFwxT85S0jQpptnyQoxcHZP2JsKPbvPI9OK7TJkHIZRBcc4vvt6atty7aj6UoY","GET_Object","sra-pub-run-8","SRR10303547/SRR10303547.1"
"bucket","storage_byte_hours"
"sra-pub-assembly-1","975588435704496"
"time_micros","c_ip","c_ip_type","c_ip_region","cs_method","cs_uri","sc_status","cs_bytes","sc_bytes","time_taken_micros","cs_host","cs_referer","cs_user_agent","s_request_id","cs_operation","cs_bucket","cs_object"
"1588633215861145","35.245.231.141","1","","GET","/storage/v1/b/sra-pub-src-9/o?prefix=SRR10123568%2FMRP2_RH_15.fq.gz.1&projection=noAcl&fields=prefixes%2CnextPageToken%2Citems%2Fname&alt=json&userProject=nih-sra-datastore&delimiter=%2F&maxResults=1000","200","0","3","29000","www.googleapis.com","","apitools gsutil/4.37 Python/2.7.13 (linux2) google-cloud-sdk/237.0.0 analytics/disabled,gzip(gfe)","AAANsUkmFIqxJGcgvTOINNI2e3TxQnhBPtfNjaupBtM3LhUBDmRUFv_4fnJMBEH-7aKOrWAo_UXYmB2IUKsf7rJpJnI","","sra-pub-src-9",""
"1588633231856193","35.199.58.46","1","","GET","/storage/v1/b/sra-pub-src-9/o?projection=noAcl&versions=False&fields=prefixes%2CnextPageToken%2Citems%2Fname&userProject=nih-sra-datastore&prefix=SRR1783158%2FFDA00008759_S4_L001_R2_001.fastq.gz&maxResults=1000&delimiter=%2F&alt=json","200","0","3","33000","www.googleapis.com","","apitools gsutil/4.37 Python/2.7.13 (linux2) google-cloud-sdk/237.0.0 analytics/disabled,gzip(gfe)","AAANsUmGoKvoGxm-mjG2m5NuGGM41ZgkdwztIX4jGUGnn6GOrfsWylHMAWbwgx9r9et7INlDZCa6aP_TIRiD5tgRj5M","","sra-pub-src-9",""
"1588633231817605","35.199.58.46","1","","GET","/storage/v1/b/sra-pub-src-9/o/SRR1783158%2FFDA00008759_S4_L001_R2_001.fastq.gz?fields=name&alt=json&userProject=nih-sra-datastore&projection=noAcl","404","0","327","27000","www.googleapis.com","","apitools gsutil/4.37 Python/2.7.13 (linux2) google-cloud-sdk/237.0.0 analytics/disabled,gzip(gfe)","AAANsUnfT4pJNPCTubzd8g1UmANNxaq3Oiuq8gSYZ4k0D0VMBiV5kh300QUYnJhiQw6L_X27NKb5JhNVR_jOM4dZ0ec","storage.objects.get","sra-pub-src-9","SRR1783158/FDA00008759_S4_L001_R2_001.fastq.gz"
"1588633238865955","35.245.222.168","1","","GET","/storage/v1/b/sra-pub-src-9/o?projection=noAcl&versions=False&fields=prefixes%2CnextPageToken%2Citems%2Fname&userProject=nih-sra-datastore&prefix=SRR10355113%2FT4.WGBS.ML.R2.fq.gz.1&maxResults=1000&delimiter=%2F&alt=json","200","0","3","32000","www.googleapis.com","","apitools gsutil/4.37 Python/2.7.13 (linux2) google-cloud-sdk/237.0.0 analytics/disabled,gzip(gfe)","AAANsUnKfEkHcjIiudX0fuMszTgpb0yBAmfSqAgfO4W2i6hDP72MNVqpoi27Ln5oSa0b2v5yvEAg2ey15lpE1iNkfWw","","sra-pub-src-9",""
"1590907585146315","130.14.28.140","1","","GET","/storage/v1/b/sra-pub-src-10/o?projection=noAcl&versions=False&fields=prefixes%2CnextPageToken%2Citems%2Fname&userProject=nih-sra-datastore&maxResults=1000&pageToken=CgtTUlIxNDkwOTA4Lw%3D%3D&delimiter=%2F&alt=json","200","0","19066","51000","storage.googleapis.com","","apitools gsutil/4.46 Python/2.7.5 (linux2) google-cloud-sdk/274.0.1 analytics/disabled,gzip(gfe)","AAANsUmRZyoGBDN4lwokbq_js0Vru4GUasypt5cdUXM95CiEHbMGAcjvn2KFyDtwS3nYMIj6P1AeqxqLmNTOR39ZV4w","","sra-pub-src-10",""
"1590903789470044","34.86.124.154","1","","GET","/storage/v1/b/sra-pub-src-10/o/SRR834799%2FDIO-probiotics_%5EAATCTACT_index.fastq?fields=name&alt=json&userProject=nih-sra-datastore&projection=noAcl","404","0","329","38000","www.googleapis.com","","apitools gsutil/4.37 Python/2.7.13 (linux2) google-cloud-sdk/237.0.0 analytics/disabled,gzip(gfe)","AAANsUlKabNGfYqgF-NxwNmUansoIaCNltf9ZUUkXaSYDV2ONWPxCMyT9269jOMOM9w5WUeaZqWg6Xsm8ZrJVRK8dzk","storage.objects.get","sra-pub-src-10","SRR834799/DIO-probiotics_^AATCTACT_index.fastq"
