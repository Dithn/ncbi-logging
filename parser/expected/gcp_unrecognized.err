converting GCP format
total : 4
accepted : 4
rejected : 0
headers : 3
unrecognized : 0
