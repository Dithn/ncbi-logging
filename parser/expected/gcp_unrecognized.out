{"accepted":true,"agent":"apitools gsutil/4.37 Python/2.7.13 (linux2) google-cloud-sdk/237.0.0 analytics/disabled,gzip(gfe)","bucket":"sra-pub-run-1","host":"www.googleapis.com","ip":"35.245.77.223","ip_region":"","ip_type":1,"method":"GET","object":"SRR002994/SRR002994.2","operation":"storage.objects.get","referer":"","request_bytes":0,"request_id":"19919634438459959682894277668675","result_bytes":0,"source":"GS","status":404,"time":1554306916471623,"time_taken":42000,"uri":"/storage/v1/b/sra-pub-run-1/o/SRR002994%2FSRR002994.2?fields=name&alt=json&userProject=nih-sra-datastore&projection=noAcl"}
{"accepted":true,"agent":"GCS Lifecycle Management","bucket":"sra-pub-src-3","host":"","ip":"","ip_region":"","ip_type":0,"method":"PUT","object":"SRR1269314/s_3_11000.p1_export.txt.gzb_fixed.bam","operation":"PUT_Object","referer":"","request_bytes":0,"request_id":"02478f5fe3deaa079d1a4f6033043a80","result_bytes":0,"source":"GS","status":200,"time":1591062496651000,"time_taken":0,"uri":""}
{"accepted":true,"agent":"","bucket":"sra-pub-run-8","host":"storage.googleapis.com","ip":"35.202.252.53","ip_region":"","ip_type":1,"method":"GET","object":"SRR10303547/SRR10303547.1","operation":"GET_Object","referer":"","request_bytes":0,"request_id":"AAANsUnhN-04LMubyLe-H4MQzIYbbqFwxT85S0jQpptnyQoxcHZP2JsKPbvPI9OK7TJkHIZRBcc4vvt6atty7aj6UoY","result_bytes":262144,"source":"GS","status":206,"time":1591118933830501,"time_taken":31000,"uri":"/sra-pub-run-8/SRR10303547/SRR10303547.1?GoogleAccessId=data-access-service%40nih-sra-datastore.iam.gserviceaccount.com&Expires=1591478828&userProject=nih-sra-datastore&Signature=ZNzj62MP4PWwSVvtkmsB97Lu33wQq4cFGLyWRJTcb%2F8h1BvVXi3lokOoT16ihScR%0At2EHti%2FgQ80VVMv9BGpAY%2FQ9HTqXeq57N53tMcjXQQMKFVttyXgIW89OLWO0UC0h%0AZdFq5AcKZywgnZql8z3RoaQi%2FPKdrdMO803tW%2Bxe%2Boy8sCd%2FyCXcG9jBrkGbdqPc%0A3xnyuycW1Va4LHIh4muGGdFSIqBk7oaLjkjLV54L8e4InzFMD3Kx0Q5raIlNadxx%0AIX%2B2hoPJuCdSh6IxEikrvUri%2Fd6i9Nqo%2BkZ%2BPSGtvlah9I9AafXrs3EAlwZkvc%2Bp%0AnjssKH8zalZQ5SmPpfHImQ%3D%3D%0A"}
{"accepted":true,"agent":"apitools gsutil/4.37 Python/2.7.13 (linux2) google-cloud-sdk/237.0.0 analytics/disabled,gzip(gfe)","bucket":"sra-pub-src-9","host":"www.googleapis.com","ip":"35.245.231.141","ip_region":"","ip_type":1,"method":"GET","object":"","operation":"","referer":"","request_bytes":0,"request_id":"AAANsUkmFIqxJGcgvTOINNI2e3TxQnhBPtfNjaupBtM3LhUBDmRUFv_4fnJMBEH-7aKOrWAo_UXYmB2IUKsf7rJpJnI","result_bytes":3,"source":"GS","status":200,"time":1588633215861145,"time_taken":29000,"uri":"/storage/v1/b/sra-pub-src-9/o?prefix=SRR10123568%2FMRP2_RH_15.fq.gz.1&projection=noAcl&fields=prefixes%2CnextPageToken%2Citems%2Fname&alt=json&userProject=nih-sra-datastore&delimiter=%2F&maxResults=1000"}
{"accepted":true,"agent":"apitools gsutil/4.37 Python/2.7.13 (linux2) google-cloud-sdk/237.0.0 analytics/disabled,gzip(gfe)","bucket":"sra-pub-src-9","host":"www.googleapis.com","ip":"35.199.58.46","ip_region":"","ip_type":1,"method":"GET","object":"","operation":"","referer":"","request_bytes":0,"request_id":"AAANsUmGoKvoGxm-mjG2m5NuGGM41ZgkdwztIX4jGUGnn6GOrfsWylHMAWbwgx9r9et7INlDZCa6aP_TIRiD5tgRj5M","result_bytes":3,"source":"GS","status":200,"time":1588633231856193,"time_taken":33000,"uri":"/storage/v1/b/sra-pub-src-9/o?projection=noAcl&versions=False&fields=prefixes%2CnextPageToken%2Citems%2Fname&userProject=nih-sra-datastore&prefix=SRR1783158%2FFDA00008759_S4_L001_R2_001.fastq.gz&maxResults=1000&delimiter=%2F&alt=json"}
{"accepted":true,"agent":"apitools gsutil/4.37 Python/2.7.13 (linux2) google-cloud-sdk/237.0.0 analytics/disabled,gzip(gfe)","bucket":"sra-pub-src-9","host":"www.googleapis.com","ip":"35.199.58.46","ip_region":"","ip_type":1,"method":"GET","object":"SRR1783158/FDA00008759_S4_L001_R2_001.fastq.gz","operation":"storage.objects.get","referer":"","request_bytes":0,"request_id":"AAANsUnfT4pJNPCTubzd8g1UmANNxaq3Oiuq8gSYZ4k0D0VMBiV5kh300QUYnJhiQw6L_X27NKb5JhNVR_jOM4dZ0ec","result_bytes":327,"source":"GS","status":404,"time":1588633231817605,"time_taken":27000,"uri":"/storage/v1/b/sra-pub-src-9/o/SRR1783158%2FFDA00008759_S4_L001_R2_001.fastq.gz?fields=name&alt=json&userProject=nih-sra-datastore&projection=noAcl"}
{"accepted":true,"agent":"apitools gsutil/4.37 Python/2.7.13 (linux2) google-cloud-sdk/237.0.0 analytics/disabled,gzip(gfe)","bucket":"sra-pub-src-9","host":"www.googleapis.com","ip":"35.245.222.168","ip_region":"","ip_type":1,"method":"GET","object":"","operation":"","referer":"","request_bytes":0,"request_id":"AAANsUnKfEkHcjIiudX0fuMszTgpb0yBAmfSqAgfO4W2i6hDP72MNVqpoi27Ln5oSa0b2v5yvEAg2ey15lpE1iNkfWw","result_bytes":3,"source":"GS","status":200,"time":1588633238865955,"time_taken":32000,"uri":"/storage/v1/b/sra-pub-src-9/o?projection=noAcl&versions=False&fields=prefixes%2CnextPageToken%2Citems%2Fname&userProject=nih-sra-datastore&prefix=SRR10355113%2FT4.WGBS.ML.R2.fq.gz.1&maxResults=1000&delimiter=%2F&alt=json"}
{"accepted":true,"agent":"apitools gsutil/4.46 Python/2.7.5 (linux2) google-cloud-sdk/274.0.1 analytics/disabled,gzip(gfe)","bucket":"sra-pub-src-10","host":"storage.googleapis.com","ip":"130.14.28.140","ip_region":"","ip_type":1,"method":"GET","object":"","operation":"","referer":"","request_bytes":0,"request_id":"AAANsUmRZyoGBDN4lwokbq_js0Vru4GUasypt5cdUXM95CiEHbMGAcjvn2KFyDtwS3nYMIj6P1AeqxqLmNTOR39ZV4w","result_bytes":19066,"source":"GS","status":200,"time":1590907585146315,"time_taken":51000,"uri":"/storage/v1/b/sra-pub-src-10/o?projection=noAcl&versions=False&fields=prefixes%2CnextPageToken%2Citems%2Fname&userProject=nih-sra-datastore&maxResults=1000&pageToken=CgtTUlIxNDkwOTA4Lw%3D%3D&delimiter=%2F&alt=json"}
{"accepted":true,"agent":"apitools gsutil/4.37 Python/2.7.13 (linux2) google-cloud-sdk/237.0.0 analytics/disabled,gzip(gfe)","bucket":"sra-pub-src-10","host":"www.googleapis.com","ip":"34.86.124.154","ip_region":"","ip_type":1,"method":"GET","object":"SRR834799/DIO-probiotics_^AATCTACT_index.fastq","operation":"storage.objects.get","referer":"","request_bytes":0,"request_id":"AAANsUlKabNGfYqgF-NxwNmUansoIaCNltf9ZUUkXaSYDV2ONWPxCMyT9269jOMOM9w5WUeaZqWg6Xsm8ZrJVRK8dzk","result_bytes":329,"source":"GS","status":404,"time":1590903789470044,"time_taken":38000,"uri":"/storage/v1/b/sra-pub-src-10/o/SRR834799%2FDIO-probiotics_%5EAATCTACT_index.fastq?fields=name&alt=json&userProject=nih-sra-datastore&projection=noAcl"}
{"accepted":true,"agent":"apitools gsutil/4.37 Python/2.7.13 (linux2) google-cloud-sdk/237.0.0 analytics/disabled,gzip(gfe)","bucket":"sra-pub-src-14","host":"www.googleapis.com","ip":"35.245.218.83","ip_region":"","ip_type":1,"method":"POST","object":"SRR1929577/{control_24h_biorep1}.fastq.gz","operation":"storage.objects.insert","referer":"","request_bytes":1467742168,"request_id":"AAANsUnxuPe3SnDN8Y2xbJ2y94VV3u924Bfq6MLxdYC5L6aemGMz3KGEFHWBlJnz96leDkMCkJZFJO-40Rw7wdV__fs","result_bytes":158,"source":"GS","status":200,"time":1588261829246636,"time_taken":11330000,"uri":"/resumable/upload/storage/v1/b/sra-pub-src-14/o?fields=generation%2CcustomerEncryption%2Cmd5Hash%2Ccrc32c%2Cetag%2Csize&alt=json&userProject=nih-sra-datastore&uploadType=resumable"}
